shared/django_apps/timeseries/migrations/0019_auto_20250206_1657.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.16 on 2025-02-06 16:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("timeseries", "0018_auto_20250206_1657"),
```

Comment:
can you give these migrations a readable name?
also, it is possible to merge some of those together, or do they have to be separate migrations?